tests/dualtor/test_orchagent_standby_tor_downstream.py

```diff
@@ -281,3 +287,62 @@ def stop_garp(ptfhost):
     tunnel_monitor.existing = True
     with crm_neighbor_checker(tor), tunnel_monitor:
         testutils.send(ptfadapter, int(ptf_t1_intf.strip("eth")), pkt, count=10)
+
+
+def test_downstream_standby_mux_toggle_active(
+    conn_graph_facts, ptfadapter, ptfhost,
+    rand_selected_dut, rand_unselected_dut, tbinfo,
+    tunnel_traffic_monitor, vmhost, toggle_all_simulator_ports
+    ):
+    # set rand_selected_dut as standby and rand_unselected_dut to active tor
+    # params = dualtor_info(ptfhost, rand_selected_dut, rand_unselected_dut, tbinfo)
+    tor = rand_selected_dut
+    test_params = dualtor_info(ptfhost, tor, rand_unselected_dut, tbinfo)
+    server_ipv4 = test_params["target_server_ip"]
+    random_dst_ip = server_ipv4
+
+    pkt = build_packet_to_server(tor, ptfadapter, random_dst_ip, tunnel_traffic_monitor)
+    exp_pkt = build_expected_packet_to_server(pkt)
+    ptf_t1_intf = random.choice(get_t1_ptf_ports(tor, tbinfo))
+
+    def set_mux_state(dut, tbinfo, state, toggle_all_simulator_ports):
```

Comment:
Can you refactor the method here to a common utility file? It also accounts for single ToR testbeds https://github.com/Azure/sonic-mgmt/blob/90c9e0e39fe4774c41ba2fec4248b1514bff7307/tests/dualtor/test_standby_tor_upstream_mux_toggle.py#L54

Reply:
Updated now, thanks!